src/comps/side-bar/sideBar.jsx

```diff
@@ -0,0 +1,177 @@
+
+import "./sideBar.css";
+
+function SideBar () 
+{
+
+  return(
+<aside className="side-bar" >
+
+
+
+<div className="filters">
+  <div className="filter-component">
+    <div className="filter-title">PRICE RANGE</div>
+
+    <div className="filter-list">
+      <label for="radio-1">
+        <input
+          type="radio"
+          name="low to high"
+          className="radio"
```

Comment:
don't write name attribute like this, [read here](https://stackoverflow.com/questions/5688200/can-i-use-white-spaces-in-the-name-attribute-of-an-html-element#:~:text=Yes.,in%20the%20middle%20is%20fine).)